build.gradle

```diff
@@ -83,6 +83,8 @@ subprojects { subproject ->
 		springRetryVersion = '1.2.2.RELEASE'
 		springVersion = '5.1.0.RC2'
 		springDataCommonsVersion = '2.1.0.RC2'
+		reactorVersion = '3.2.0.M4'
+		reactorKafkaVersion = '1.0.1.RELEASE'
```

Comment:
Let's consider to update these dependencies to see if the API is better now!

Reply:
Done